doc/sg/c_cpp_coding_style.md

```diff
@@ -55,20 +55,36 @@ if (foo) {
   do_something();
 }
 ```
+### Infinite loops
+
+Prefer `while(true){}` infinite loops rather than `for(;;)`
 
 ### Comments
 
-***Comments should be `// C99-style` for consistency with C++.***
+Comments should use the `// C99-style` for consistency with C++.
 
 <!-- To render a backtick in inline code in markdown, you need to double the surrounding backticks.
 https://daringfireball.net/projects/markdown/syntax#code -->
-***Variables mentioned in comments should be delimited with backtick (`` ` ``) characters.***
-
+Variables mentioned in comments should be delimited with backtick (`` ` ``) characters.
 Example:
 
 ```c
 // `ptr` can never be NULL for reasons.
 ```
+Documentation comments should start with `/*` and non-doc comments should start with `//`.
+Example:
+
+```c
+/**
+ * This function sorts an int array very quickly.
+ */
+void sort(int array[], size_t size){
+  // Loop through the array moving higher numbers to the end.
+  for(size_t i = 0; i < size; i++>){
```

Comment:
Should this be `++i` following: https://google.github.io/styleguide/cppguide.html#Preincrement_and_Predecrement

```suggestion
  for (size_t i = 0; i < size; ++i) {
```

Reply:
Fixed